file-manager/src/main/scala/it/pagopa/interop/commons/files/service/FileManager.scala

```diff
@@ -42,9 +42,9 @@ trait FileManager {
 
   def close(): Unit
 
-  def generateGetPresignedUrl(bucketName: String, path: String, fileName: String): Try[String]
+  def generateGetPresignedUrl(bucketName: String, path: String, fileName: String, duration: Int): Try[String]
```

Comment:
`duration` should be of type `FiniteDuration`, or leaving as `Int` is ok, but I suggest renaming it to `durationInMinutes`